src/System.Management.Automation/engine/ParameterBinderController.cs

```diff
@@ -345,7 +345,7 @@ internal static void AddArgumentsToCommandProcessor(CommandProcessorBase command
                             {
                                 param = CommandParameterInternal.CreateParameterWithArgument(
                                     /*parameterAst*/null, paramText.Substring(1, colonIndex - 1), paramText,
-                                    /*argumentAst*/null, paramText.Substring(colonIndex + 1).Trim(),
+                                    /*argumentAst*/null, paramText.AsSpan(colonIndex + 1).Trim().ToString(),
```

Comment:
ToString() here results in an allocation, can we just update that method to accept a readonlyspan?

Reply:
No, it is used then in ctor-s.

Reply:
`ReadOnlySpan<char>` typically is an implicit cast from `string`, isn't it?

Reply:
@vexx32 No, only string -> ReadOnlySpan<char>

Reply:
Right, so if the method parameter is currently using `string` we should be able to change it to ReadOnlySpan<char> and have the implicit cast handle it, shouldn't we? Or am I missing some context here? 🙂

Reply:
>  I missing some context here

CreateParameterWithArgument() passes the string to a constructor so it makes no sense to convert to span here.